vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -53,9 +53,28 @@ This file content adds two fields to the Conference entity, introduces the Sessi
 ```
 ent jhipster import-jdl conference.jdl
 ```
-If the default project structure has been retained, this step will update your data model, add entries to Liquibase to upgrade database schema during deployment, add service methods to your microservice, add fields to your MFEs, etc.
 
-5. You can now build your updated project and [run it locally](./run-local.md) or [deploy it to Entando](../pb/publish-project-bundle.md). Definition enhancement through build and test can be repeated as many times as needed.
+If the default project structure has been retained, this step updates your data model, adds entries to Liquibase to upgrade database schema during deployment, adds service methods to your microservice, adds fields to your MFEs, etc.
+  - For pre-existing micro frontends, they need to be moved because they have been regenerated. From the project directory:   
+
+	```shell  
+	mv microservices/conference-ms/ui/widgets/conference/tableWidget/{.,}* microfrontends/conference-table
+	```  
+
+  - For new micro frontends, use the ent CLI to add them to the bundle descriptor, relocate them, and add an API claim to extablish the connection. From the project directory:
+      1. Add the new MFE and move it to the `microfrontends/YOUR-NEW-MFE` folder:
+      ```shell
+      ent bundle mfe add YOUR-NEW-MFE  
+      mv microservices/conference-ms/ui/widgets/conference/YOUR-NEW-MFE/{.,}* microfrontends/YOUR-NEW-MFE
+	  ```
+      2. Add an API claim to connect the new MFE to the pertinent microservice. If this is an extension of the Blueprint generated project, an API claim should connect the new MFE to the `conference-ms` microservice.
+
+	  ```
+	  ent bundle api add YOUR-NEW-MFE conference-api --serviceName=conference-ms --serviceUrl=http://localhost:8081
```

Comment:
This command should be run from the bundle root folder where we can find the entando.json too

Reply:
Because this instruction follows step 1, and there is no change in environment, it can be assumed they are still in the same bundle root directory.  But reworded lines 58 and 64 for clarity.

Reply:
Ok